packages/app/src/systems/Send/components/Send/Send.test.tsx

```diff
@@ -31,7 +31,6 @@ describe('Send', () => {
     it('should show transaction details by default', async () => {
       render(<Select />, { wrapper: TestWrapper });
       await waitFor(() => findAssetSelect());
-      expect(await screen.findByText('Fee (network)')).toBeInTheDocument();
```

Comment:
Should we remove this?

Reply:
maybe we should remove this whole test as we have it in e2e already. fixed it for now